examples/MultipleRedeemers.purs

```diff
@@ -0,0 +1,254 @@
+import Contract.Prim.ByteArray (byteArrayFromAscii)
+import Contract.ScriptLookups as Lookups
+import Contract.Scripts (Validator, validatorHash, MintingPolicy)
+import Contract.Transaction
+  ( BalancedSignedTransaction(BalancedSignedTransaction)
+  , balanceAndSignTx
+  , submit
+  )
+import Contract.TxConstraints as Constraints
+import Contract.Utxos (utxosAt)
+import Contract.Value (mkTokenName, scriptCurrencySymbol)
+import Contract.Value as Value
+import Contract.Wallet (mkNamiWalletAff)
+import Data.Array (replicate)
+import Data.BigInt (fromInt)
+import Data.Foldable (length, sum)
+import Data.Bitraversable (bitraverse)
+import Data.Either (hush)
+import Data.Int (toNumber)
+import Data.Map as Map
+import Data.Set as Set
+import Data.Maybe (Maybe)
+import Data.Traversable (traverse)
+import Data.Tuple (Tuple(Tuple), fst, snd)
+import Effect.Aff (Aff, delay, Milliseconds(Milliseconds), error)
+import Effect.Aff.Class (liftAff)
+import Plutus.ToPlutusType (toPlutusType)
+import Plutus.Types.Address (scriptHashAddress)
+import Plutus.Types.Transaction (UtxoM, Utxo)
+import Plutus.Types.CurrencySymbol (getCurrencySymbol)
+import Types.Scripts (ValidatorHash)
+import Types.TokenName (TokenName)
+import Types.Transaction (TransactionHash, TransactionInput)
+
+-- FIXME: this doesn't work without a browser
+threeRedeemerContract :: Aff Unit
+threeRedeemerContract = do
+  log "creating utxos"
+  ContractConfig defaults <- traceContractConfig
+  wallet <- mkNamiWalletAff
+  let
+    cfg :: ContractConfig ()
+    cfg = ContractConfig $ defaults { wallet = pure wallet }
+
+  validator :: Validator <- liftM (error "Invalid script JSON")
+    alwaysSucceedsScript
+  vhash :: ValidatorHash <- liftM (error "could not hash validator") $
+    validatorHash validator
+
+  Tuple hash numTokens :: Tuple TransactionHash Int <-
+    runContract cfg $ do
+
+      let
+        -- FIXME: Try to mint 3 tokens
+        tokens :: Array (Tuple String Int)
+        tokens =
+          [ Tuple "pleasedontspent" 3
+          ]
+
+      mp :: MintingPolicy <- liftContractM
+        "could not decode MintingPolicy from Json"
+        alwaysMintsPolicy
+      cs :: CurrencySymbol <-
+        liftContractM "could not get curencysymbol from mintingpolicy" $
+          mkCurrencySymbol <<< getCurrencySymbol =<< scriptCurrencySymbol mp
+      -- TODO: It would probably make sense to incorporate an actual Script instead of the alwayssucceeds 
+      --       one as someone could always just spend our tokens while we wait between the transactions
+      tok :: Array (Tuple TokenName Int) <-
+        liftContractM "could not make tokennames with values" $ traverse
+          (bitraverse (mkTokenName <=< byteArrayFromAscii) pure)
+          tokens
+
+      logInfo' $ "trying to create" <> show tok
+
+      let
+        toCsValue :: Array (Tuple TokenName Int) -> Value.Value
+        toCsValue t =
+          unwrap <<< toPlutusType <<< Value mempty <<< NonAdaAsset
```

Comment:
You should use the Plutus-style `Value` from `Contract.Value` instead. There's no need to use the Cardano ones, unless I'm missing something (and even if this is necessary at the moment, that's a bug that needs to be fixed -- we don't want users to have to think about the Cardano vs. Plutus thing)

Reply:
Got it. Currently I'm searching the documentation - pursuit doesn't tell me which modules reexport the things I need hence I'm just taking the module they're defined in.

Reply:
I see, unfortunately we can't do anything about that AFAIK. For the examples, we should be writing them as a user would. So we should also only use `Contract.*` for CTL stuff

Reply:
I think it would be sensible to just export a lot of the really commonly used stuff from the Contract.Prelude. Then you wouldn't have to worry about that and which types to import.

Reply:
So this is still an issue - I can not directly construct a Plutus Value, also I could not purge all the non-Contract imports.

Reply:
> So this is still an issue - I can not directly construct a Plutus Value

There's a Monoid instance and a `singleton` function, which should be enough to construct anything. However, if you can think of other helper functions missing, please share.